frontend/public/components/routes/create-route.tsx

```diff
@@ -637,7 +637,7 @@ const AlternateServicesGroup: React.FC<AlternateServiceEntryGroupProps> = (props
 
   React.useEffect(() => {
     props.onChange({ name, weight }, props.index);
-  }, [name, weight]);
+  }, [name, props, weight]);
```

Comment:
I'd pull this out of your pr and deal with it separately since it's unrelated

Reply:
I think this was a lint fix

Reply:
Right, it's only a warning, though. I'd rather deal with it separately. It would be better to destructure that prop.

Reply:
I opened #3046